3rdparty/lpg_planner/Makefile

```diff
@@ -0,0 +1,26 @@
+all: installed
+
+INSTALL_DIR=`rospack find lpg_planner`
+FILENAME = lpg1.2-linux.tar.gz
+TARBALL = build/$(FILENAME)
+TARBALL_URL = http://github.com/jsk-ros-pkg/archives/raw/master/$(FILENAME)
```

Comment:
Compared to https://github.com/jsk-ros-pkg/jsk_3rdparty/blob/master/3rdparty/julius/Makefile.grammar-kit ,
the software version should be fixed.

Reply:
ok, I add version